File: week1/02-mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Ask for height
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0);
    for (int i = 0; i < height; i++)
    {
        // Print spaces
        for (int j = 1; j < height - i; j++)
        {
            printf(" ");
        }

        // Print #
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: week6/03-sentimental-mario-more/mario.py
from cs50 import get_float, get_int, get_string


def main():
    mario()


def mario():
    levels = get_int("Height: ")
    while levels <= 0 or levels > 8:
        levels = get_int("Height: ")
    for i in range(1, levels + 1):
        print(f"{' ' * (levels - i)}{'#' * (i)}  {'#' * (i)}")


main()
